Lib/traceback.py

```diff
@@ -826,7 +826,7 @@ def __eq__(self, other):
     def __str__(self):
         return self._str
 
-    def format_exception_only(self):
+    def format_exception_only(self, show_group=False, depth=0):
```

Comment:
I'd consider making these keyword-only:

```suggestion
    def format_exception_only(self, *, show_group=False, depth=0):
```